### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by mckent05**
In this pull request, my partner (@ntazelo) and I have added tests for the add and delete functions in my to-do List project. we have made use of the Jest library to test the required functions and ensure that they meet all the necessary project requirements.

### CODE REVIEW

File: src/add_list.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//   test('add task to to-do List', () => {
+//     expect(addTask(taskList, 'samuel', false, 0)).toHaveLength(1);
+//     expect(addTask(taskList, 'tope', false, 1)).toHaveLength(2);
+//     expect(addTask(taskList, 'tope', false, 2)).toHaveLength(3);
+//   });
+//   test('remove task from to-do', () => {
+//     expect(deleteToDo(taskList, 'samuel')).toHaveLength(2);
+//   });
+// });
+
```
Comment (GzimAsani) [inline src/add_list.test.js:15]: - [ ] Please remove the commented out code

Comment (GzimAsani) [reply src/add_list.test.js:15]: - [ ] Update your readme file and let the users know how can they test your project

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @mckent05  and @ntazelo                              
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the gitflow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @mckent05 & @ntazelo,

You've done well addressing the previous review concerns.
Your project is now complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**